### PR TITLE
Added entry for travis-ci

### PR DESCRIPTION
**Pull request by nmacuk**
I guess I configured travis, did not do "git rebase" but made a branch from a repo that was already up to date with master, is it okay?
I think we have a problem with ruby versions because we did not specify one and may have different on our local machines.

### CODE REVIEW

**Comment by nmacuk** [comment]
`$ — bundle install — jobs=3 — retry=3 — bundle exec rake db:create — bundle exec rake db:migrate — bundle exec rspec
—: command not found
The command "— bundle install — jobs=3 — retry=3 — bundle exec rake db:create — bundle exec rake db:migrate — bundle exec rspec" exited with 127.`

This is what travis-ci site and logs tell me. I have no idea what's wrong with the configuration's file syntax

**Comment by nmacuk** [comment]
`$ bundle exec rails db:migrate`
`rails aborted!`
`PG::ConnectionBad: could not connect to server: No such file or directory
	Is the server running locally and accepting
	connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"?`

there is a problem with a db, no idea really what's wrong. I guess because it has something to do with the DB server and that as I assume postgress needs, not like SQLite. Am I right?

**Comment by nmacuk** [comment]
Finally done it. Needed to changed database.yml file in order to use different database and postgres. Added build status to readme as well.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (PiotrMisiurek) [inline LICENSE]: This file is not a part of this travis

File: README.md [hunk 2]
```diff
@@ -1,24 +1,26 @@
-# README
+# Unnamed-Rails-Project
```
Comment (PiotrMisiurek) [inline README.md]: this file is not a part of travis feature

File: config/database.yml [hunk 3]
```diff
@@ -57,8 +57,8 @@ development:
 # Do not set this db to the same as development or production.
 test:
   <<: *default
-  database: unnamedrailsproject_test
-
+  adapter: postgresql
+  database: travis_ci_test
```
Comment (PiotrMisiurek) [inline config/database.yml]: those changes should not be required

File: README.md (hunk 2 above)
Comment (nmacuk) [reply README.md]: You said that readme part with build status should be a part of Travis pr...

File: LICENSE (hunk 1 above)
Comment (nmacuk) [reply LICENSE]: Well I guess it was pulled from master somehow and got here. I'll delete it

**Comment by nmacuk** [comment]
Changed Rubocop and travis configuration, disabled some cops, fixed some previous offenses. I guess so far it's all we needed.

**Comment by nmacuk** [comment]
> Is Travis doing the same thing twice - once for branch, once for PR?

Changed it now in Travis's settings.

**Comment by PiotrMisiurek** [comment]
> > Is Travis doing the same thing twice - once for branch, once for PR?
> 
> Changed it now in Travis's settings.

Can you repush it to confirm that it works?

**Comment by nmacuk** [comment]
>     1. Repush branch to origin to confirm that it is doing its job only one
> 
>     2. This branch/PR request should include only changes related directly to travis. Make rubocop pass is another thing. Please at least rebase commits to have one including all travis related changes and another with style correction in files

Changed it to only Branch and repushed, did not split the branch into another for rubocop and travis because I guess I made too many bad commented commits and I can't really track where the changes were made. I'm sorry.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+before_script:
+  - psql -c 'create database db_test;' -U postgres
+rvm: 
+  - "2.6.0"
+script:
+  - bundle install
+  - bundle exec rails db:create
+  - bundle exec rails db:migrate
+  - bundle exec rake spec
+  - bundle exec rubocop -a
```
Comment (PiotrMisiurek) [inline .travis.yml]: There is no point of running rubocop with `-a` flag. The autocorrected changes won't be commited. The goal is to see if check is clear or not.

**Pull request closed without merging**